scripts/ort-evaluator.sh

```diff
@@ -11,16 +11,49 @@ else
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+if [[ -s "$ORT_CONFIG_LICENSE_CONFIGURATION_FILE" ]]; then
+    ORT_EVALUATOR_OPTIONS="--license-classifications-file $ORT_CONFIG_LICENSE_CONFIGURATION_FILE"
+else
+    ORT_EVALUATOR_OPTIONS=""
+fi
+
+if [[ -d "$ORT_CONFIG_PACKAGE_CONFIGURATION_DIR" ]]; then
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-dir $ORT_CONFIG_PACKAGE_CONFIGURATION_DIR"
+elif [[ -s "$ORT_CONFIG_PACKAGE_CONFIGURATION_FILE" ]]; then
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-file $ORT_CONFIG_PACKAGE_CONFIGURATION_FILE"
+elif [[ -d "$ORT_CONFIG/packages" ]]; then
+    # Use legacy named package configuration directory if present.
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-dir $ORT_CONFIG/packages"
+    ORT_CONFIG_PACKAGE_CONFIGURATION_DIR=$ORT_CONFIG/packages
+fi
+
+if [[ ! -s "$ORT_CONFIG_RULES_FILE" ]]; then
+    # Use legacy named policy rules file if present
+    if [[ -s "$ORT_CONFIG_DIR/rules.kts" ]]; then
+        ORT_CONFIG_RULES_FILE="$ORT_CONFIG_DIR/rules.kts"
+    else
+        # Use first found file with .rules.kts extension 
+        FIND_RESULTS=$(find $ORT_CONFIG_DIR -type f -name '*.rules.kts')
+        RESULT_FILES=( $FIND_RESULTS )
+        ORT_CONFIG_RULES_FILE="${RESULT_FILES[0]}"
+    fi
+fi
+
+if [[ -s "$ORT_CONFIG_RULES_FILE" ]]; then
+    echo "Using ORT policy rules file found in $ORT_CONFIG_RULES_FILE..."
+else
+    echo "Error: ORT policy rules file (*.rules.kts) not found. Please set ORT_CONFIG_RULES_FILE"
+fi
+
 $ORT \
     --$ORT_LOG_LEVEL \
     --stacktrace \
     evaluate \
+    $ORT_EVALUATOR_OPTIONS \
     -i $ORT_RESULTS_INPUT_FILE \
     -o $ORT_RESULTS_DIR \
-    -r $ORT_CONFIG_DIR/rules.kts \
-    -f JSON \
-    --license-classifications-file $ORT_CONFIG_LICENSE_CONFIGURATION_FILE \
-    --package-configuration-dir $ORT_CONFIG_PACKAGE_CONFIGURATION_DIR
+    -r $ORT_CONFIG_RULES_FILE \
```

Comment:
out of scope: Maybe rename to singular `RESULTS` in `ORT_RESULTS_INPUT_FILE` and `ORT_RESULTS_DIR`.